targets/targets.json

```diff
@@ -2052,7 +2052,6 @@
             "LPC"
         ],
         "detect_code": ["0236"],
-        "device_name": "LPC55S69JBD100",
         "release_versions": ["5"]
```

Comment:
```suggestion
        "sectors": [[0, 32768]],
        "release_versions": ["5"]
```

Reply:
i'm unable to move the suggestion to the _NS target...please do not commit this in the base target